paddle/operators/deconv2d_op.cc

```diff
@@ -0,0 +1,100 @@
+/* Copyright (c) 2016 PaddlePaddle Authors. All Rights Reserve.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License. */
+
+#include "paddle/operators/deconv2d_op.h"
+#include "paddle/operators/conv2d_op.h"
+
+namespace paddle {
+namespace operators {
+
+void Deconv2DOp::InferShape(framework::InferShapeContext* ctx) const {
```

Comment:
From the comments: https://github.com/tensorflow/tensorflow/issues/256#issuecomment-162257789

How about rename Conv2DTranspose?

Reply:
Great suggestion!